### PR TITLE
Update main.cpp

### PR DESCRIPTION
**Pull request by kevin-koshy**
Slack username:kevin-koshy

Creating pull request for Project 1

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun:Keyboard
+//  action 1 Types Characters
+keyboard.typeCharacters();
+//  action 2: Control the computer
+keyboard.controlComputer();
+//  action 3:costs Money
+keyboard.costMoney();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

File: main.cpp [hunk 2]
```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun:lightBulb    
+//  action 1:Spreads Light
+lightBulb.spreadLight();
+//  action 2:Genates Heat
+lightBulb.generateHeat();
+//  action 3:Costs Money
+lightBulb.costMoney
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: missing `();`

**Review by matkatmusic — CHANGES_REQUESTED** [review]
you wrote `costMoney()` as an action for 7 out of the 10 exercises.
Try to be more creative on these projects.

**Comment by kevin-koshy** [comment]
Updated main.cpp after review

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**